dynamic-demo-plugin/oc-manifest.yaml

```diff
@@ -87,8 +87,11 @@ spec:
     port: 9001
     basePath: '/'
   proxy:
-    services:
-      - name: thanos-querier
-        namespace: openshift-monitoring
-        port: 9091
-        authorize: true
+  - type: Service
+    alias: demoplugin
+    name: thanos-querier
```

Comment:
So the `ConsoleProxyService` does not contain `name` field. Check https://github.com/openshift/api/blob/master/console/v1alpha1/types_console_plugin.go#L46-L92
What @spadgett was referring to was that the `alias` field should be `thanos-querier`